dagger-core/src/main/java/io/odpf/dagger/core/processors/external/es/EsSourceConfigBuilder.java

```diff
@@ -0,0 +1,108 @@
+package io.odpf.dagger.core.processors.external.es;
+
+import io.odpf.dagger.core.processors.common.OutputMapping;
+
+import java.util.Map;
+
+public class EsSourceConfigBuilder {
```

Comment:
do we really need this class? It's only being used in the test? If we need this we can move it to test package?

Reply:
Yes we need this class, it makes the code much more readable. given in this pr we are focusing on refactoring tests, wanted to make minimal changes to actual code. will start using the builder class in main code afterwards.